image_encode.py

```diff
@@ -0,0 +1,30 @@
+"""
+For encoding images returned by the camera device.
+"""
+import cv2
+import numpy as np
+
+def encode_image_as_bytes(ext: str, image: "np.ndarray") -> "tuple[bool, bytes | None]" :
```

Comment:
Rename this to `encode_image` or `image_encode` . The `as_bytes` is redundant with the return type.